### ISSUE TITLE
Feature request/question: and 'ends with' option to chant text search page? 

### ISSUE THREAD

**Issue by kyrieb-ekat** [DDMAL/CantusDB#1640]
Not sure if this has already been discussed or not, but happy to talk about it if not! I'm curious if it would be possible to add an option for searching for chant texts- currently we have 'contains' and 'begins with'. Possible to add 'ends with'? \

Granted, this is maybe a niche use case for fragments, or a poor digitization, but just a thought...

**Comment by annamorphism** [comment]
this is something you can do for chant texts on Cantus Index...(I think we had the same list of search functions on CantusDB at one time and decided to only have it one place? but maybe it wasn't really a conscious decision. Or I'm misremembering.)

**Comment by dchiller** [comment]
It's a very easy thing to add technically, so we can implement whatever the decision is.

**Comment by kyrieb-ekat** [comment]
Might be nice to have it on the cantusdb end without having to go to Cantus Index to make those searches, is my thinking 😁

**Comment by dchiller** [comment]
Agreed that it seems like a lot to expect that people would know to go to Cantus Index...

Just to confirm, you're looking for something like the following: 

"This is a chant that I am singing"

The search "am singing" or "singing" would match but "that I" or "I am" would not. Right?

**Comment by kyrieb-ekat** [comment]
Yes!

**Comment by kyrieb-ekat** [comment]
April25: confirmed!
